build-system/tasks/check-links.js

```diff
@@ -146,6 +146,10 @@ function filterWhitelistedLinks(markdown) {
   filteredMarkdown =
       filteredMarkdown.replace(/(\(|\[)?http:\/\/localhost:8000/g, '');
 
+  // cdn.ampproject.org link
```

Comment:
Whitelisting is a bad idea, as this is meant to represent a domain and not a a clickable link. You can edit the `.md` file and put `https://cdn.ampproject.org` within backticks to bypass the link checker.